docs/reference/query-dsl/exists-query.asciidoc

```diff
@@ -70,3 +70,23 @@ no values in the `user` field and thus would not match the `exists` filter:
 { "foo": "bar" }
 --------------------------------------------------
 
+===== `missing` query
+
+The exists query can advantageously replace the `missing` query when used inside a must_not clause as follows:
+
+[source,js]
+--------------------------------------------------
+"constant_score": {
+    "filter": {
+        "bool": {
+            "must_not": {
+                "exists": {
+                    "field": "user"
+                }
+            }
+        }
+    }
+}
+--------------------------------------------------
```

Comment:
I don't think we need the outer constant_score>filter, let's just provide users with the `bool` part?